lib/paper_trail/model_config.rb

```diff
@@ -196,6 +204,8 @@ def setup_callbacks_from_options(options_on = [])
       options_on.each do |event|
         public_send("on_#{event}")
       end
+
+      public_send("on_touch")
```

Comment:
I think you can just call `on_touch` here, right? The `public_send` is not necessary.